<?xml version="1.0"?>
<launch>

    <arg name="panda_name" default="panda"/>
    <arg name="base_name" default="$(arg panda_name)/base"/>
 
    <group ns="$(arg panda_name)">

        <node pkg="cooperative_manipulation_controllers" type="franka_impedance_control_sim.py" name="ts_control_sim_only" output="screen">
            <param name="base" value="$(arg base_name)"/>
        </node>

    </group>  

</launch>

